conf/server.yaml.template

```diff
@@ -8,12 +8,10 @@ SERVER:
     # RELEASE: populate with satellite version
     # The snap version currently testing (if applicable)
     # SNAP:
-    # The RHEL Base OS Version(x.y) where the Satellite is installed
-    # RHEL_VERSION:
     # The source of Satellite packages. Can be one of:
     # internal, ga, beta
     SOURCE: "internal"
-    RHEL_RELEASE: '7'
```

Comment:
This is referenced in the template for repos.yaml and needs to be updated to be consistent.

https://github.com/SatelliteQE/robottelo/blob/8cba71e33b2755306cbb93d447ab50c12c9e2542/conf/repos.yaml.template#L5

Reply:
> This is referenced in the template for repos.yaml and needs to be updated to be consistent.
> 
> https://github.com/SatelliteQE/robottelo/blob/8cba71e33b2755306cbb93d447ab50c12c9e2542/conf/repos.yaml.template#L5

I did update it but missed in check-in. Updated.